frappe/database/query.py

```diff
@@ -501,27 +575,67 @@ def set_fields(self, fields, **kwargs):
 		fields.extend(function_objects)
 		return fields
 
+	def join_(self, criterion, fields, table, join):
```

Comment:
joining seems to be wrong from what I can see

<img width="840" alt="Screenshot 2022-08-30 at 1 38 43 PM" src="https://user-images.githubusercontent.com/32034600/187385012-9c8eddf3-36e7-43f6-9216-0126e68aa288.png">

<img width="843" alt="Screenshot 2022-08-30 at 1 39 06 PM" src="https://user-images.githubusercontent.com/32034600/187384993-e4ded247-6566-4eeb-b92f-00c378b5f44b.png">

Reply:
`ch` is a `Table` field in `inty` doctype

Reply:
Int names also need to be handled while joining

Reply:
I think implicit joins are broken in QB it self considering removing it for now 🤔

Reply:
removed implicit joins [55afe2](https://github.com/frappe/frappe/pull/17537/commits/c55afe23f80969be2af905c79f0b3cb90d992896)